src/datasource/datasource.test.js

```diff
@@ -0,0 +1,170 @@
+import { DataSource } from "./datasource";
+
+const testConfig = {
+  "gatewayUrl": "http://localhost:3000/v1/60676c9f7cbbfe002f0b9cbe",
+  // "gatewayUrl": "https://mainnet.gateway.pokt.network/v1/60676c9f7cbbfe002f0b9cbe",
+  "http": {
+    "timeout": 0,
+    "headers": {
+      "Content-Type": "application/json",
+      "Blockchain-Subdomain": "mainnet",
+    }
+  },
+}
+
+describe('DataSource', () => {
+  const context = {
+    datasource: null,
+    args: {
+      getAccount: [
+        "04c56dfc51c3ec68d90a08a2efaa4b9d3db32b3b",
+      ],
+      getTransaction: [
+        "E4A3EDE68171473996E68549C5CFC3C06B4865C35D194B8BDDA08908B4D394A6",
```

Comment:
Bit of a nitpick, but I'd put these two very specific vars at the top of the file and in uppercase, something like `POKT_ACCOUNT` and `POKT_TRANSACTION` (depending on the context obviously.

Reply:
Love it! thanks!